### PR TITLE
Updated Slack join link

### LINKED ISSUE #6: Slack invite link invalid and hardcoded
Hi,

it seems that the slack invite link is invalid. (Also found on the PROD site)
https://join.slack.com/t/hpcng/shared_invite/zt-gy0st6mt-ijgUaSvfdeEOhfXXfIstrQ

In addition to that it seems that the value is hardcoded in the `index.html` file but not parameterized or similar. (e.g. https://github.com/rocky-linux/rockylinux.org/blob/f6ff1e105eca27fbba1467d128518f1e6b9c4387/public/index.html#L159)

Cheers,
Elys

### PR DESCRIPTION
**Pull request by zeeshanjamal16**
The old slack join link was expired. Replaced it with the new slack join link.

Fixes #6

### CODE REVIEW

**Comment by Addvilz** [comment]
@zeeshanjamal16 please, comment on the relevant task OR create one BEFORE doing something. 

You are missing the link in several files - footer, 404, etc. Please grep the whole source base for slack.com and replace the link everywhere.

**Comment by zeeshanjamal16** [comment]
@Addvilz 
Sure. I wiil keep in mind.
I have updated the link in other files as well.

**Review by Addvilz — APPROVED** [review]

File: public/index.html [hunk 1]
```diff
@@ -68,10 +68,10 @@ <h2 class="text-4xl tracking-tight font-display leading-10 font-extrabold text-g
...
                 </div>
             </div>
-        </main>
+        </main>https://join.slack.com/t/hpcng/shared_invite/zt-k5z04bsh-1uqpaD1NsYVP73vzc3uKdQ
```
Comment (danieltharp) [inline public/index.html]: ?

**Review by danieltharp — CHANGES_REQUESTED** [review]

**Comment by hbjydev** [comment]
Would you be able to store the invite link as a variable in the `config.toml` to ensure that we never have to go searching for every instance of `slack.com` again?

An example of how to reference `config.toml` values can be found in the `layouts/partials/meta/site-twittercard.html` file.

File: public/index.html (hunk 1 above)
Comment (zeeshanjamal16) [reply public/index.html]: Sorry. My bad. Just removed it.

**Comment by zeeshanjamal16** [comment]
@hbjydev 
Updated hard coded links to config.toml. Can you please check the same?

**Review by Addvilz — APPROVED** [review]

**Review by danieltharp — APPROVED** [review]

**Pull request merged**